### ISSUE TITLE
UI links to CLI are broken

### ISSUE THREAD

**Issue by superseb** [rancher/rancher#15978]
**Rancher versions:**
rancher/server or rancher/rancher: v2.1.0

i.e. for Linux:
```
https://releases.rancher.com/cli/V2.0.5/rancher-linux-amd64-V2.0.5.tar.gz
```

Seems that CLI is published under `/cli2/` and we have an uppercase typo (`V` vs `v`)

```
https://releases.rancher.com/cli2/v2.0.5/rancher-linux-amd64-v2.0.5.tar.gz
```

**Comment by loganhz** [comment]
I can download the cli via Rancher UI

I guess it's not a typo in the commit https://github.com/rancher/rancher/commit/32e12d5e1a418f917dc9b170e3354bf4e28c803b
. 

@alena1108 Can you confirm it, please?

**Comment by vincent99** [comment]
I copied them to where it's looking on the CDN so it wouldn't be broken anymore, but it's still the wrong place.

**Comment by alena1108** [comment]
@loganhz it is a typo and @superseb fix is legit

**Comment by loganhz** [comment]
Fixed in 2.1.3
